webapp-django/crashstats/crashstats/static/crashstats/css/screen.less

```diff
@@ -830,7 +830,7 @@ div.code {
     float: right;
     .current {
         float: right;
-        background-image: url(../../../img/3rdparty/fatcow/stop16x16.png);
+        background-image: url(../../img/3rdparty/fatcow/stop16x16.png);
```

Comment:
I don't understand this change. Was the original url wrong and the fixed url right? What was the error this was kicking up?

Reply:
The error happened during asset compilation; it couldn't resolve the path to an existing collected static file. It's unclear to me why this worked before and didn't work after, though. I couldn't find any Django or django-pipeline changes that seemed to be related.

I don't actually see any instances of "oopp-hang" in the codebase. Maybe this is just unused and that's why we didn't notice it, and something changed to make the failure no longer silent?